src/cli.ts

```diff
@@ -37,6 +50,37 @@ const main = async (argv: string[]) => {
       });
     });
 
+  program
+    .command("create-tasks")
+    .description(
+      "create tasks for reviewing the pre-translations of this branch"
+    )
+    .addOption(
+      new BranchNameOption(
+        "the Crowdin branch for which to create tasks"
+      ).makeOptionMandatory(true)
+    )
+    .addOption(
+      new Option(
+        "-f, --file-id <number>",
+        "file ID for which to create the tasks"
+      ).makeOptionMandatory(true)
+    )
```

Comment:
Would this require the engineer to manually enter the file ID when running the command? Alternatively we could maybe add this as an option (eg `--create-tasks`) for the `upload` command? Then you’d have access to the file ID within the handler.